test/e2e_flags.go

```diff
@@ -73,9 +82,48 @@ func initializeFlags() *EnvironmentFlags {
 
 	flag.StringVar(&f.Tag, "tag", "latest", "Provide the version tag for the test images.")
 
+	klog.InitFlags(nil)
+	flag.Set("v", "2") // A normal level of logging for testing
```

Comment:
I thought this would silently fail

Reply:
I mistaken why changes weren't having any effect; it wasn't having effect because of the level (v) not because of a flag being Set here. I'll move alsologtostderr up.

Reply:
If that's the case I would suggest move alsologtostderr back to serving

Reply:
It has to be called after klog.InitFlags, so I wouldn't trust it anywhere else.